### PR TITLE
Simple calc

### PR DESCRIPTION
**Pull request by srleach**
## Features

- Add flexible finance calculator to basket

Delivers [#121944781](https://www.pivotaltracker.com/story/show/121944781)

### CODE REVIEW

**Comment by srleach** [comment]
@00eadeyemi Can you look over this please chief?

File: app/Http/Middleware/VerifyCsrfToken.php [hunk 1]
```diff
@@ -14,6 +14,7 @@ class VerifyCsrfToken extends BaseVerifier
      */
     protected $except = [
         'push/*',
+        'ajax/*',
```
Comment (wnowicki) [inline app/Http/Middleware/VerifyCsrfToken.php]: - why? I think @EvanBarbour3 fixed that in past

Comment (srleach) [inline app/Http/Middleware/VerifyCsrfToken.php]: I did see that, actually. @EvanBarbour3 can you explain what you did?

**Comment by sped2773** [comment]
- Holiday and term in months to h4
- deposit color is default not ain blue

**Comment by sped2773** [comment]
Pre-review complete and tested. @wnowicki for final review.

**Pull request merged**